tests/PHPStan/Rules/Generics/data/bug-9153.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace ClxProductNet\Recombee\Model;
+
+/**
+ * @template-covariant Value of string|list<string>
+ *
+ * @immutable
```

Comment:
I think this needs another test-case with a `@readonly` property on a non-immutable class